bot/skills/__init__.py

```diff
@@ -36,16 +35,27 @@ def _add_version(upd: Updater, version_handlers_group: int):
                                   filters=admin_filter), version_handlers_group)
 
 
+def _get_version_from_toml() -> str:
+    """ Parse toml file for version """
+    toml_file_content = open('pyproject.toml', 'r').read()
```

Comment:
Did you close the file? Looks like no! Call `toml_file_content.close()` or use `open` like context manager (better way):

```python
with open('pyproject.toml', 'r') as pyproject:
    toml_dict = toml.loads(pyproject.read())
```

https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files

Reply:
Fixed.